modules/twinklexfd.js

```diff
@@ -21,7 +21,7 @@ Twinkle.xfd = function twinklexfd() {
 	if ( mw.config.get('wgNamespaceNumber') < 0 || !mw.config.get('wgArticleId') || (mw.config.get('wgNamespaceNumber') === 6 && (document.getElementById('mw-sharedupload'))) ) {
 		return;
 	}
-	Twinkle.addPortletLink( Twinkle.xfd.callback, "XFD", "tw-xfd", "Nominate for deletion" );
+	Twinkle.addPortletLink( Twinkle.xfd.callback, "XFD", "tw-xfd", "Nominate for deletion discussion" );
```

Comment:
Disagree with this. We're nominating the page to be considered for _deletion_, not nominating it to be considered for a _deletion discussion_, which anyone can start unilaterally.
```suggestion
	Twinkle.addPortletLink( Twinkle.xfd.callback, "XFD", "tw-xfd", "Nominate for deletion" );
```

Reply:
Only AfD and MfD are explicitly XfDeletion; all the rest are XfDiscussion.  Deletion is clearly the main/common thrust, hence this "compromise."  I don't love the line(s), maybe "Nominate ''to'' deletion discussion''?  ''at''?  "Start a deletion discussion"?

Reply:
Given that it can close as merge/redirect, even if its called XfDeletion, maybe "Create a new deletion discussion" or the "Start a deletion discussion" proposed above, but I agree that we are not nominating it for deletion discussion.

Reply:
Mmm, yeah, I like `Create new deletion discussion` and `Start a deletion discussion`.  Leaning toward the latter.

```suggestion
	Twinkle.addPortletLink( Twinkle.xfd.callback, "XFD", "tw-xfd", "Start a deletion discussion" );
```